projects/canopy/src/styles/shadow.scss

```diff
@@ -1,3 +1,7 @@
 .lg-shadow {
   box-shadow: var(--drop-shadow);
 }
+
+.lg-shadow-hover {
```

Comment:
Should this be applied on hover only? so the selector be `.class-selector:hover`?

Reply:
Thanks, Elena. I've fixed this now.